pkgs/tools/audio/openai-whisper-cpp/default.nix

```diff
@@ -9,16 +9,26 @@
 , CoreML
 , CoreVideo
 , MetalKit
+
+, config
```

Comment:
```suggestion
```

Reply:
Without `config`, I have `error: undefined variable 'config'` because of the `config.cudaSupport` of the next line.